SpatialGDK/Source/SpatialGDK/Private/Interop/WellKnownEntitySystem.cpp

```diff
@@ -130,4 +135,51 @@ void WellKnownEntitySystem::InitializeVirtualWorkerTranslationManager()
 	VirtualWorkerTranslationManager = MakeUnique<SpatialVirtualWorkerTranslationManager>(Receiver, Connection, VirtualWorkerTranslator);
 	VirtualWorkerTranslationManager->SetNumberOfVirtualWorkers(NumberOfWorkers);
 }
+
+void WellKnownEntitySystem::MaybeClaimSnapshotPartition()
+{
+	// Perform a naive leader election where we wait for the correct number of server workers to be present in the deployment, and then
+	// whichever server has the lowest server worker entity id claims the snapshot partition.
```

Comment:
super nit 
```suggestion
	// whichever server has the lowest server worker entity ID becomes the leader and claims the snapshot partition.
```
Read a little as if the leader election was unrelated